lib/swaggerUtil.js

```diff
@@ -106,7 +106,8 @@ function validateObject(specs, obj, disc) {
     else if (!specs.definitions[obj[disc]]) {
         validationErrors.push({
             message: 'model definition not found for ' + disc +
-                ': ALL model definitions must be defined in root swagger document'
+                ': ALL model definitions must be defined in root swagger document. ' +
+                'This can be accomplished by turning ref compilation "on"'
```

Comment:
I think this message makes it too simple ... there are implications to turning it on (such as BOS overwriting the entire `definitions` section of the spec), and it won't always help, it all depends on how they've written their spec.
Also, there's configuration necessary for the ref compiler to work, so it's not as easy as just turning it "on".

At best, I think we can include a message like:
`The built-in $ref compiler can be used to manage your definitions for you.\nLearn more at <link-to-relevant-wiki-page>`